sys/statvfs.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * statvfs
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Lukasz Kosinski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <sys/file.h>
+#include <sys/msg.h>
+#include <sys/stat.h>
+#include <sys/statvfs.h>
+
+
+int statvfs(const char *restrict path, struct statvfs *restrict buf)
+{
+	char *canonical;
+	oid_t oid, dev;
+	msg_t msg;
```

Comment:
msg.o.data, msg.o.size, msg.i.data and msg.i.size are never set - this will cause kernel to try to map some random address during msgSend
```suggestion
	msg_t msg = { 0 };
```

Reply:
Fixed.